KamerConnect.View.MAUI/Pages/ProfilePage.xaml.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using KamerConnect.Models;
+using KamerConnect.Services;
+using KamerConnect.Utils;
+using KamerConnect.View.MAUI.Views;
+
+
+namespace KamerConnect.View.MAUI.Pages;
+
+public partial class ProfilePage : ContentPage
+{
+    private readonly FileService _fileService;
+    private readonly PersonService _personService;
+    private readonly AuthenticationService _authenticationService;
+    private const string _bucketName = "profilepictures";
```

Comment:
Als deze op 2 plekken staat zoals die nu staat zal ik die op 1 plek zetten als config. Dit kan mischien ook in de local.env dat je een PROFILE_PICTURE_BUCKET daarin aanmaakt